### ISSUE TITLE
Site notifications are broken

### ISSUE THREAD

**Issue by juho-jaakkola** [juho-jaakkola/elgg-chat#7]
@Pasley70 said:

> The recipients site notification - on opening a new chat - has no title and link to the chat, but just pointing "A new Chat" with a "broken" icon. It would be useful, if the recipient get the same information on the site as in the E-Mail. Maybe a conflict with new site notification in 1.9?

**Comment by juho-jaakkola** [comment]
I'm not sure if site notifications are needed at all. After all the chat icon in the topbar notifies user about unread chat messages. I think I'd prefer disabling site notifications completely.

What do you think?

**Comment by Pasley70** [comment]
Yes, you are right. Site-Notifications are redundant. I agree with your proposal.
